core/templates/services/user-backend-api.service.ts

```diff
@@ -34,29 +34,20 @@ export interface EmailPreferencesBackendDict {
   'can_receive_subscription_email': boolean;
 }
 
-interface NonEmailPreferencesBackendDict {
+export interface NonEmailPreferencesBackendDict {
   'preferred_language_codes': string[];
   'preferred_site_language_code': string;
   'preferred_audio_language_code': string;
-  'profile_picture_data_url': string;
   'default_dashboard': string;
   'user_bio': string;
   'subject_interests': string;
+}
+
+export interface ProfilePictureDataBackendDict {
```

Comment:
Why is this together with `subscription_list`?

Reply:
Not any reason. I am moving it to `PreferencesHandler`

Reply:
done